config.py

```diff
@@ -199,6 +201,11 @@
             "KubernetesServices": "AKS",
         },
     },
+    "firebase": {
```

Comment:
Put the `firebase` key below the `gcp` key to keep the order consistency with other dict and tuples.

Reply:
I Fixed it.